generator_test.go

```diff
@@ -472,6 +473,38 @@ func makeTestNewV7Basic() func(t *testing.T) {
 	}
 }
 
+// makeTestNewV7TestVector as defined in Draft04
+func makeTestNewV7TestVector() func(t *testing.T) {
+	return func(t *testing.T) {
+		pRand := make([]byte, 10)
+		//TODO make the comparison work with
```

Comment:
Need to reconcile this TODO before thinking about merging this.

Reply:
I will remove the TODO but I failed to do the actual validation on the random data compared with the example from the draft. At least for now I think a partial validation is better than nothing (the test now only asserts the first 15bytes).